gn3/api/ctl.py

```diff
@@ -1,22 +1,24 @@
 """module contains endpoints for ctl"""
 
-# so small 
-
 from flask import Blueprint
 from flask import request
 from flask import jsonify
 
 from gn3.computations.ctl import call_ctl_script
 
-ctl = Blueprint("ctl",__name__)
+ctl = Blueprint("ctl", __name__)
+
 
-@ctl.route("/run_ctl",methods=["POST"])
+@ctl.route("/run_ctl", methods=["POST"])
 def run_ctl():
-	"""endpoint to run ctl"""
-	ctl_data = request.json
+    """endpoint to run ctl
+    input: request form object
+    output:json object enum::(response,error)
 
-	results = call_ctl_script(ctl_data)
+    """
+    ctl_data = request.json
 
-	return jsonify({
-	"results":results
-	})
\ No newline at end of file
+    (cmd_results, response) = call_ctl_script(ctl_data)
+    return (jsonify({
+        "results": response
+    }), 200) if response is not None else (jsonify({"error": str(cmd_results)}), 401)
```

Comment:
**issue:**  Just create an extra variable here to make things more readable.  Prefer readability over brevity/ conciseness.